src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp

```diff
@@ -1922,7 +1922,7 @@ GUI_TEST_CLASS_DEFINITION(test_6277) {
                       .arg(EXPECTED_COLOR_COLUMN_WIDTH)
                       .arg(colorColumnWidth));
 
-    CHECK_SET_ERR(static_cast<double>(colorColumnWidth) / totalTableWidth >= 0.2,
+    CHECK_SET_ERR(static_cast<double>(colorColumnWidth) / totalTableWidth >= 0.13,
```

Comment:
Looks like we do not need this and other "width" related changed after you removed min-width change for Options Panel

Reply:
That's right, reverted